mmv1/products/firebase/terraform.yaml

```diff
@@ -54,6 +54,15 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           org_id: :ORG_ID
         ignore_read_extra:
           - project
+          - deletion_policy
```

Comment:
We may want to call out something about importing the field in the docs, i.e.

> Upon import, this field will always take on the default value of `"ABANDON"`, regardless of what's specified in configuration. If you have another value set, a diff will appear in `terraform plan`. Run `terraform apply` to apply that value.

Reply:
this seems like not a great experience - would this be possible for us to fix / do we already have a ticket open to fix it?

Reply:
https://github.com/hashicorp/terraform-plugin-framework/issues/62